index.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Why DOcumentin in Python</title>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="css/style.css">
+
+<style>
+ @media (max-width: 600px) {
+    body {
+        background-color: #f2f0eb;
+    }
+}
+</style>
```

Comment:
Please move this media query to the external stylesheet  "style.css" located in the folder css. It  will reinforce the capability of your code to be maintained.